### PR TITLE
Document in CONTRIBUTING.md how to run tests and devcards

### PR DESCRIPTION
**Pull request by roryokane**
Devcards instructions were based on https://github.com/athensresearch/athens/pull/109#issuecomment-636395081

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,48 +4,120 @@ Not convinced you want to learn Clojure? Read this developer's [first month expe
...
+
+[Devcards](https://github.com/bhauman/devcards) are pages that show just one component of the web app, for the purpose of demonstrating or testing how that component looks when rendered with certain data.
+
+To open this project’s devcards:
+
+1.  Instead of `lein dev`, start a local server like this:
+
+    ```
+    lein devcards
+    ```
```
Comment (jeroenvandijk) [inline CONTRIBUTING.md:50]: You can also view devcards when you run `lein dev`. I realized this later after Jeff said that. The url below is correct. When you run `lein dev` and `lein devcards` at the same time, the url will actually be different, e.g. 3001. So maybe best to not mention `lein devcards` for newbees

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,48 +4,120 @@ Not convinced you want to learn Clojure? Read this developer's [first month expe
...
+
+2.  Open http://localhost:3000/cards.html.
 
-# Clojure Style Guide
+3.  (optional) Using your fork, run `lein gh-pages` to see your own branch, e.g.
+    https://tangjeff0.github.io/athens/cards.html
```
Comment (teodorlu) [inline CONTRIBUTING.md:55]: What does this actually do? Build assets? Build assets and push to repo on a separate branch? It's using Github pages, right?

Do I have to "activate Github pages" on my fork for this to work?

**Comment by teodorlu** [comment]
@roryokane I think this looks great!

- Clear, concise writing
- Common questions adressed.

I made a (small) comment with questions about what the `lein gh-pages` command actually does.

Thanks! :100:

File: CONTRIBUTING.md (hunk 2 above)
Comment (jeroenvandijk) [reply CONTRIBUTING.md:55]: @teodurlu From the top of my head, I did it once (for the sci.web), I only had to follow these instructions https://github.com/borkdude/sci.web#release . No need to turn something on

I'm guessing `lein gh-pages` is executing something like that

**Pull request merged**